static/js/index.js

```diff
@@ -1,6 +1,16 @@
 function renderGitHubCommits() {
   let updated = "";
   latestCommits.forEach((commit) => {
+    // first create a date object
+    const date = new Date(commit["date"]+" UTC");
+    var options = { year: 'numeric', 
+                      month: 'long', 
+                      day: 'numeric' , 
+                      hour: "2-digit" , 
+                      minute: "2-digit" , 
+                      second: "2-digit"
+                    };
+    const strDate = date.toLocaleDateString("en-US", options)
```

Comment:
You probably could just use the default options here, I don't think you need to specify hour/minute/second specifically.

Reply:
I just committed with default options, and then realized that it doesnt actually display time unless you put this info here. in other words, default is display nothing. so I added it back